pkg/sbproxy/reconcile/reconcile_visibilities.go

```diff
@@ -63,28 +62,41 @@ func (r *resyncJob) brokerNames(brokers []*platform.ServiceBroker) []string {
 	return names
 }
 
-func (r *resyncJob) getSMPlansByBrokersAndOfferings(ctx context.Context, offerings map[string]*types.ServiceOffering) (map[string][]*types.ServicePlan, error) {
+func (r *resyncJob) getSMBrokerPlans(
+	ctx context.Context, offerings map[string]*types.ServiceOffering, smBrokers []*platform.ServiceBroker) (
```

Comment:
Do you really insist the arguments and return type to be on separate lines? It's not that longer than most other functions.

Reply:
ok, this is now the longest line in this file at 172 chars

It's inconvenient to scroll sideways, but probably this should be addressed separately for the whole project